app/controllers/admin/dashboard_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Admin::DashboardController < Admin::BaseController
+
+  def index
+    @item = Item.new
+    @items = Item.all
+    @category =Category.all
+    @user = User.find(session[:user_id])
+  end
```

Comment:
We should consider changing these to links to pages that we can do this on.

Reply:
It doesn't look like these are being used in the view. So I'm going to remove them.